data/net/minecraft/world/level/levelgen/carver/WorldCarver.mapping

```diff
@@ -1,34 +1,64 @@
 CLASS net/minecraft/world/level/levelgen/carver/WorldCarver
 	METHOD <init> (Lcom/mojang/serialization/Codec;)V
 		ARG 1 codec
+	METHOD canReach (Lnet/minecraft/world/level/ChunkPos;DDIIF)Z
+		ARG 0 chunkPos
+		ARG 1 x
+		ARG 3 z
+		ARG 5 branchIndex
+		ARG 6 branchCount
+		ARG 7 width
 	METHOD canReplaceBlock (Lnet/minecraft/world/level/block/state/BlockState;)Z
 		ARG 1 state
 	METHOD canReplaceBlock (Lnet/minecraft/world/level/block/state/BlockState;Lnet/minecraft/world/level/block/state/BlockState;)Z
 		ARG 1 state
 		ARG 2 aboveState
 	METHOD carve (Lnet/minecraft/world/level/levelgen/carver/CarvingContext;Lnet/minecraft/world/level/levelgen/carver/CarverConfiguration;Lnet/minecraft/world/level/chunk/ChunkAccess;Ljava/util/function/Function;Ljava/util/Random;Lnet/minecraft/world/level/levelgen/Aquifer;Lnet/minecraft/world/level/ChunkPos;Ljava/util/BitSet;)Z
+		COMMENT Carves the given chunk with caves that originate from the given {@code chunkPos}.
+		COMMENT This method is invoked 289 times in order to generate each chunk (once for every position in an 8x8 chunk radius centered around the target chunk).
```

Comment:
I think you mean 17x17? since 17^2 is 289 and the code iterates from -8 to 8 on both axes.

Reply:
Yeah, I think I was trying to write both "17x17 area" and "8 chunk radius" and somehow merged the two xD

Reply:
ahh lol